app/assets/javascripts/client/directives/learning_questions/learningQuestionModal.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .directive('learningQuestionModal', learningQuestionModal);
+
+  function learningQuestionModal() {
+    return {
+      restrict: 'E',
+      scope: {
+        reminder: '@'
+      },
+      templateUrl: 'client/views/modals/learning_questions/add_learning_question.html',
+      link: function(scope) {
+        // The modal exists on the parent scope, so we have to reach up to it  to bind any closing actions on the modal.
+        var parentScope = scope.$parent;
```

Comment:
Emitting the event would be a fair bit tougher to track down and debug.  Here, we're making it explicit that the parent contains the modal, and that we require the parent scope to do so.